src/rgw/rgw_d3n_cacherequest.h

```diff
@@ -29,7 +25,7 @@ class D3nCacheRequest {
     off_t len;
     std::string key;
     off_t read_ofs;
-    Context *onack;
+    Context* onack;
```

Comment:
might be good to namespace-qualify Context

Reply:
oh, it's unused - removing